cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -35,3 +35,13 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# On Windows there is no support for @rpath/LD_LIBRARY_PATH so we need to set PATH to find cocotb libraries
+# https://docs.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order?redirectedfrom=MSDN#standard-search-order-for-desktop-applications
+export PATH := $(LIB_DIR):$(PATH)
```

Comment:
Can we just put this inside the `ifeq ($(OS),Msys)` in `Makefile.inc`? It looks unrelated to python, so doesn't seem to belong in this file.

Reply:
BTW: We can remove `pylib`s files in general also `Makefile.inc`

Reply:
Same with `LIB_DIR` so ...

Reply:
We can't, because `.pylib` is for users who copied the endian example, as I remarked  in https://github.com/cocotb/cocotb/pull/1588#discussion_r404159930.

For that reason, we should not be adding anything new to it, unless it's needed for those users.